### PR TITLE
Documentation improvement

### PR DESCRIPTION
**Pull request by cbernat**
- Added title in docs with link to style guide.  (Doppler UI Library)
image

A link to documentation in style guide will be provided in next layout. 
- Added documentation link in readme.

https://cdn.fromdoppler.com/doppler-ui-library/build484/documentation/

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -155,7 +162,7 @@ gulp.task('doc', function() {
 
 // Generated Sassdoc to Dist folder
 gulp.task('doc:dist', function() {
-  return gulp.src(config.folderAssets.base + '/**/*.scss')
+  return gulp.src('/' + config.folderAssets.base + '/**/*.scss')
```
Comment (andresmoschini) [inline gulpfile.js]: How does it affect the result?

Comment (cbernat) [reply gulpfile.js]: This is not supposed to be part of the change, I will remove it, thanks!

**Comment by andresmoschini** [comment]
Something happens...

**Comment by cbernat** [comment]
> Something happens...
> 
> image

Yes sure it does, adding an extra bar does not work.

**Pull request merged**